core/src/ConfirmationModal.html

```diff
@@ -14,7 +14,7 @@ <h3 class="fd-title fd-title--h5">{settings.header}</h3>
         </div>
       </div>
     </header>
-    <div class="fd-message-box__body">{settings.body}</div>
+    <div class="fd-message-box__body">{@html settings.body}</div>
```

Comment:
i can't find any sanity check for settings.body, so this allows to inject anything into core app